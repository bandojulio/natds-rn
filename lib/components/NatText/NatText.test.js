var _jsxFileName='lib/components/NatText/NatText.test.js';var _react=require('react');var _react2=_interopRequireDefault(_react);
var _reactTestRenderer=require('react-test-renderer');var _reactTestRenderer2=_interopRequireDefault(_reactTestRenderer);
require('jest-styled-components/native');

var _NatText=require('./NatText');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}

describe('NatText component default',function(){
it('Should render NatText component ',function(){
var button=_reactTestRenderer2.default.create(
_react2.default.createElement(_NatText.NatText,{value:'Teste',__source:{fileName:_jsxFileName,lineNumber:10}})).
toJSON();

expect(button).toMatchSnapshot('NatText snapshot');
});
});
//# sourceMappingURL=NatText.test.js.map