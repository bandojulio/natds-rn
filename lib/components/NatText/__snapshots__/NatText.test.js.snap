// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NatText component default Should render NatText component  1`] = `
<View
  defaultMargin="0px"
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "marginTop": 0,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#333333",
          "fontSize": 20,
          "fontWeight": "400",
          "lineHeight": 24,
        },
      ]
    }
    type="h6"
  >
    Teste
  </Text>
</View>
`;
